### ISSUE TITLE
Build automation

### ISSUE THREAD

**Issue by Macklehatton** [TriNetCo/NetGrid#3]
I've mentioned before that there aren't a lot of options for doing this with Unity but it's worth looking into if we're talking about treating this as an extendable platform.

**Comment by TheNotary** [comment]
Yes, I've been clicking through some stuff in the GH Actions marketplace for Unity build scripts thinking someone might have produced something re-usable already.  No luck yet so I might aim for lower hanging fruit before I knuckle down in this area.
